platform/server/http/Proxy/Proxy.ts

```diff
@@ -0,0 +1,84 @@
+import { Server } from 'net';
+import { IncomingMessage, OutgoingMessage } from 'http';
+import { EventEmitter } from 'events';
+
+import { Logger } from '../../../logging';
+
+export class Proxy extends EventEmitter {
```

Comment:
It's not really a "proxy" - it's more like a "fallback", so just semi-randomly picked "proxy".

Reply:
`ProxyToLegacyPlatform`, just to be clearer?

Reply:
✔️